examples/console/ConsoleApp/Program.cs

```diff
@@ -1,39 +1,41 @@
 ﻿using System;
-using System.Numerics;
 using System.Threading;
 using Box2DX.Collision;
+using Box2DX.Common;
 using Box2DX.Dynamics;
 
-namespace ConsoleApp1
+namespace ConsoleApp
 {
     public class Program
     {
         private static void Main()
         {
-            var w = new World(
+            var world = new World(
                 new AABB()
                 {
-                    LowerBound = new Vector2(-100, -100),
-                    UpperBound = new Vector2(100, 100)
+                    LowerBound = new Vec2(-100, -100),
+                    UpperBound = new Vec2(100, 100)
                 },
-                new Vector2(0, -10),
+                new Vec2(0, -10),
                 false);
             var groundDef = new BodyDef();
-            groundDef.Position = new Vector2(0, -10);
+            groundDef.Position = new Vec2(0, -10);
 
-            var ground = w.CreateBody(groundDef);
+            var ground = world.CreateBody(groundDef);
             ground.SetStatic();
 
             var groundFixtureDef = new EdgeDef();
-            groundFixtureDef.Vertex1 = new Vector2(-100, 0);
-            groundFixtureDef.Vertex2 = new Vector2(100, 0);
+            groundFixtureDef.Vertex1 = new Vec2(-100, 0);
+            groundFixtureDef.Vertex2 = new Vec2(100, 0);
             ground.CreateFixture(groundFixtureDef);
 
             var ballDef = new BodyDef();
-            ballDef.Position = new Vector2(0, 0); Body ball = w.CreateBody(ballDef);
+            ballDef.Position = new Vec2(0, 0);
+
+            var ball = world.CreateBody(ballDef);
 
             var ballFixtureDef = new CircleDef();
-            ballFixtureDef.LocalPosition = new Vector2(0, 0);
+            ballFixtureDef.LocalPosition = new Vec2(0, 0);
```

Comment:
Vec2 is deprecated. You should get warnings requesting that it's upgraded to System.Numerics.Vector2.

Reply:
ConsoleApp uses the old code (version 1.0.3), but in this version there is no Vector2.

Reply:
https://github.com/benzuk/box2d-netstandard/blob/master/src/ConsoleApp1/Program.cs

Take a look at namespaces. It's master branch.

Reply:
With newer code, this should be Box2D.NetStandard.*. I was confused with Vector2, so I referenced the project to version 1.0.3 (nuget package).